tests/lightning_base/test_device.py

```diff
@@ -570,7 +570,7 @@ def test_preprocess_correct_config_setup(self, config, expected_config):
         assert new_config == expected_config
 
     @pytest.mark.skipif(
-        device_name="lightning.tensor", reason="lightning.tensor does not support adjoint"
+        device_name == "lightning.tensor", reason="lightning.tensor does not support adjoint"
```

Comment:
It's so weird that we don't get an error with that.

Reply:
:100: @LuisAlfredoNu @josephleekl does it mean that we skipped this test for all the devices? :thinking: